sdk/cosmos/azure-cosmos/azure/cosmos/_execution_context/aio/base_execution_context.py

```diff
@@ -119,6 +119,9 @@ async def _fetch_items_helper_no_retries(self, fetch_function):
         fetched_items = []
         # Continues pages till finds a non-empty page or all results are exhausted
         while self._continuation or not self._has_started:
+            # For start time option, First fetch will always return empty
+            # before we get a continuation token
+            first_fetch = not self._has_started
```

Comment:
Would `has_started` be true for a Change Feed Pull that is starting off a continuation?

Reply:
No, it will always start as false when a change feed pull is called, regardless of options.